openstates/md/committees.py

```diff
@@ -14,25 +14,33 @@ def scrape(self, term, chambers):
         doc.make_links_absolute(url)
 
         for a in doc.xpath('//a[contains(@href, "cmtepage")]'):
-            url = a.get('href').replace('stab=01', 'stab=04')
-            chamber = a.xpath('../../..//th/text()')[0]
+            url = a.get('href')
+            chamber_name = a.xpath('../../..//th/text()')[0]
+            if chamber_name == 'Senate Standing':
+                url = url.replace('stab=01', 'stab=04')
+            if chamber_name == 'House Standing':
+                url = url.replace('stab=01', 'stab=04')
             com_name = a.text
             if com_name is None:
                 continue
+            com_name = com_name.replace("Joint ", "")
+            com_name = com_name.replace("Committee on ", "")
+            com_name = com_name.replace(" Committee", "")
```

Comment:
Might also be worth uppercasing the first letter of the committee name after these replacements, in case it was a preposition and used to be lower-case.